CI: Makefile make snapshot fail

**What happened**:
```
make snapshot

docker run --rm --privileged \
	-e REVISIONDATE=2026-07-20 \
	-e PRIVATE_KEY= \
	-v ~/go/pkg/mod:/go/pkg/mod \
	-v `pwd`:/go/src/github.com/juicedata/juicefs \
	-v /var/run/docker.sock:/var/run/docker.sock \
	-w /go/src/github.com/juicedata/juicefs \
	juicedata/golang-cross:v1.25.7-0 release --snapshot --clean --skip-publish
Unable to find image 'juicedata/golang-cross:v1.25.7-0' locally
v1.25.7-0: Pulling from juicedata/golang-cross
f2520f145cbb: Pull complete
954d6059ca7b: Pull complete
9c31f7713c41: Pull complete
cac7b480234d: Pull complete
c141f9961fad: Pull complete
0fdf71b47847: Pull complete
9420814830c6: Pull complete
7596952c92ff: Pull complete
403a10736f1a: Pull complete
c2203b9f0601: Pull complete
b517322ff27b: Pull complete
8d414aead394: Pull complete
b5e2021c4c8b: Pull complete
80613573d3ab: Pull complete
18c011b9cedd: Pull complete
7cdedb1d6630: Pull complete
e1722ee99515: Pull complete
ad1276d6b2f3: Pull complete
4f4fb700ef54: Pull complete
16eda42c652f: Pull complete
98a05b78b1cb: Pull complete
01715698f2dd: Pull complete
d3b37a8f89c9: Pull complete
92808c1cc443: Pull complete
8aeba0471564: Pull complete
3d71b3db2e8b: Pull complete
37d08847ed91: Pull complete
1756e866f9b0: Pull complete
04155e612069: Pull complete
Digest: sha256:ee655196e10c42c0c60d7b16df07ea8a1c1e8bd258de5aa2e9523234bfa02f68
Status: Downloaded newer image for juicedata/golang-cross:v1.25.7-0
can not find any private key, ignore...
  ⨯ command failed                                   error=unknown flag: --skip-publish
make: *** [Makefile:88：snapshot] 错误 1
```
**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**

**Environment**:
- JuiceFS version (use `juicefs --version`) or Hadoop Java SDK version:
- Cloud provider or hardware configuration running JuiceFS: master branch
- OS (e.g `cat /etc/os-release`): Ubuntu 26.04 LTS
- Kernel (e.g. `uname -a`): Linux pc 7.0.0-15-generic 15-Ubuntu SMP PREEMPT_DYNAMIC Wed Apr 22 16:06:43 UTC 2026 x86_64 GNU/Linux
- Object storage (cloud provider and region, or self maintained): self maintained s3
- Metadata engine info (version, cloud provider managed or self maintained): mysql8.0
- Network connectivity (JuiceFS to metadata engine, JuiceFS to object storage): Intranet Network
- Others:

--- a/Makefile
+++ b/Makefile
@@ -92,7 +92,7 @@ snapshot:
 		-v `pwd`:/go/src/github.com/juicedata/juicefs \
 		-v /var/run/docker.sock:/var/run/docker.sock \
 		-w /go/src/github.com/juicedata/juicefs \
-		juicedata/golang-cross:v1.25.7-0 release --snapshot --clean --skip-publish
+		juicedata/golang-cross:v1.25.7-0 release --snapshot --clean
 
 release:
 	docker run --rm --privileged \

